runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -489,3 +489,30 @@ PROTO_CMD?=protoc
 SOLINK_SLINK+=protobuf ssl
 
 CXX_DEFINES+=GOOGLE_PROTOBUF_NO_RTTI
+
+ifneq ($(OPENSSL_CFLAGS),)
+    SET_FLAG=false
```

Comment:
Can we name this variable something like `ADD_OPENSSL_CFLAGS ` to make it more clear what it's for?